client/tos/index.js

```diff
@@ -0,0 +1,32 @@
+/* global wcpay_tos_settings */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import TosModal from './modal';
+import shotTosNotice from './disabled-notice';
+
+// eslint-disable-next-line camelcase
+if ( wcpay_tos_settings.tosAgreementRequired ) {
+	renderTosModal();
+}
+
+// eslint-disable-next-line camelcase
+if ( wcpay_tos_settings.tosAgreementDeclined ) {
+	window.addEventListener( 'load', () => {
```

Comment:
I know that we should wait for load event to make sure wc admin notes data layer is loaded but let's add a comment to make it clear for others as well.

Reply:
Note added: https://github.com/Automattic/woocommerce-payments/pull/993/commits/22030a5b638f4c7cf5c7039adf070fc3f7cab333#diff-861d8b0d68e919447cfdba6251c53a22f778524529c6778e70fdc5f93394817fR19